components/front-page/Topics.tsx

```diff
@@ -73,35 +80,39 @@ export const Topics = ({
           }}
           slides={chunk(topics, 3).map((topics, index) => (
             <div className="flex gap-2">
-              {topics.map((topic, index) => (
-                <Link
-                  href={`/topics/${topic.slug}`}
-                  key={index}
-                  className="flex flex-1 cursor-pointer items-center gap-4 rounded-lg bg-white p-2 transition-all hover:bg-gray-200 hover:bg-opacity-30 md:max-w-sm"
-                >
-                  <div className="relative h-10 w-10">
-                    <Image
+              {topics.map((topic, index) => {
+                return (
+                  topic.slug !== "gaza" && (
```

Comment:
I think you meant `memecoin-markets` here instead of `gaza`, but anyways... I don't think it's optimal to exclude the `memecoin-markets` here in this `Topics` component, because [this line](https://github.com/zeitgeistpm/ui/blob/7aae53bbda6edd0c35fc4e8d3a31d462be170ed3/pages/index.tsx#L176) indicates that it will select the first entry from the CMS, which is the memecoin-markets entry. I would rather wait for the access of the CMS instead of doing the dirty fix like here. When will you get access to the CMS? If it took too long to gain access to the CMS, I would suggest changing [this line](https://github.com/zeitgeistpm/ui/blob/7aae53bbda6edd0c35fc4e8d3a31d462be170ed3/pages/index.tsx#L176) to skip to the next item from the CMS if the `memecoin-markets` topic is going to be selected.

Reply:
yes, thank you. haven't touched Zeitgeist code for a while so just getting back to it. I saw your comment from earlier on DIscord and have removed the sections as per your suggestion